src/Pages/jiyeon/Login/Login.js

```diff
@@ -1,13 +1,78 @@
-import React, { Component } from 'react';
-
-class Login extends Component {
-    render() {
-        return (
-            <div>
-                
-            </div>
-        );
-    }
-}
+import React from "react";
+import { Link } from 'react-router-dom';
+import facebook_logo from '../../../images/jiyeon/facebook_icon.png'
+import './Login.scss';
+
+class LoginJY extends React.Component {
+  constructor(){
+    super();
+    this.state = {
+      idInput: '',
+      pwInput: '',
+      isBtnActive: false
+    };
+  }
+
+  handleIdPwInput = (event) => {
+    this.setState({
+      [event.target.name]: event.target.value
+    });
+  }
 
-export default Login;
\ No newline at end of file
+  handleBtnActive = () => {
+    this.setState({
+      isBtnActive: this.state.idInput.indexOf('@') !== -1 && this.state.pwInput.length >= 5 ? true : false
```

Comment:
indexOf를 사용하신 이유가 궁금합니다! : )

Reply:
indexOf를 사용하면 @가 있을 시 해당 위치를 반환하고, 없을 시에는 -1을 반환해서 -1이 아닐 경우를 확인하기 위해서 썼습니다!

Reply:
해당 코드도 위 리뷰와 같이 구조분해할당 해주시면 가독성에 더 좋겠네요! :)

Reply:
지연님! 질문 주신 내용과 함께 답변드리도록 하겠습니다. 
- 말씀하신대로 해당 코드는 굳이 '삼항연산자' 조건문으로 작성하지 않으셔도 됩니다.
아래와 같이 조건만 작성 후 해당 코드의 불리안값을 이용하시면 되겠습니다. 그러나 지연님이 작성하셨던 코드는 "!"이 들어가서 지연님께서 원하셨던 조건이 해당되지 않았던것 같네요.
- 더불어 아래와 같이 조건문을 상수에 할당하여 사용하시면 가독성에 좋은 코드가 될 수 있습니다.
```suggestion
     const { idInput, pwInput } = this.state; 
     const btnCondition = idInput.indexOf('@') !== -1 && pwInput.length >= 5;
      isBtnActive: btnCondition
```

Reply:
신영님! @bonbon0202 하기와 같이 코드 수정 완료했습니다!
저렇게 코드를 수정하고 btnCondition값을 console.log를 하였을때 true 값이 이전과 동일하게  5개 입력하였을때 바로 true로 변하지 않고 6개 입력시에 ture로 변화가 됩니다.
5개 입력했을때 바로 안되는 것이 setState로 변경이되면 render가 되어서 바로 ture로 변경되어야 하는데 모르겠습니다..
this.setState로 isBtnActive: btnCondition 을 넣어주고 콜백함수로 this.handleIdPwValue를 실행해도 결과는 같습니다..
```
  handleBtnActive = () => {
    const { idValue, pwValue } = this.state;
    const btnCondition = idValue.includes("@") && pwValue.length >= 5;
    
    this.setState({
      isBtnActive: btnCondition 
    });
  };
```